app/controllers/application_controller.rb

```diff
@@ -1200,11 +1200,20 @@ def set_config(db_record) # pass in the db record, either @host or @vm
       # Add disks to the device array
       unless db_record.hardware.disks.nil?
         db_record.hardware.disks.each do |disk|
-          loc = disk.location.nil? ? "" : disk.location
-          dev = disk.controller_type ? disk.controller_type << " " << loc : ""  # default device is controller_type
-          desc = disk.filename                              # default description is filename
-          icon = disk.device_name                       # default icon prefix is device_name
+          # relying on to_s to force nils into ""
+          loc = disk.location.to_s
```

Comment:
You're right, it's not needed anymore, but is there any harm in keeping it consistently stringy?